src/go.mod

```diff
@@ -1,3 +1,249 @@
-module github.com/onflow/api-service/m/v2
-
-go 1.18
+module github.com/onflow/api-service/m/v2
+
+go 1.18
```

Comment:
Is there a reason we're using 1.17 in the Makefile and 1.18 here?

Reply:
Yes, there was a build error that is elaborated in the Dockerfile. Probably the best idea is to stick to 1.17, as some libs are still non-compatible with 1.18.